python_benchmarks/core.py

```diff
@@ -8,6 +8,7 @@
 def get_device_properties() -> tuple[int, float]:
```

Comment:
Given you are not sure which Python versions, I think it's better to `from typing import Tuple; Tuple[int, float]`